### PR TITLE
Use Redux in React components.

### PR DESCRIPTION
**Pull request by sajid-munawar**
**In this milestone**
- Understand the concept of state management.
- Use store, actions, and reducers in React.
- Connect React and Redux.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -2,16 +2,23 @@
...
 
+export const bookList = [
+  { title: 'Design Patterns', author: 'Erich Gamma', id: '12' },
+  { title: 'Patterns of Enterprise Application Architecture', author: ' Martin Fowler', id: '13' },
+  { title: 'Code Complete', author: 'Steve Mcconnell', id: '14' },
+  { title: 'Enterprise Integration Patterns', author: 'Gregor Hohpe', id: '15' },
+];
 // Reducer
-const booksReducer = (state = [], action) => {
+// eslint-disable-next-line default-param-last
```
Comment (ahmetbozaci) [inline src/redux/books/books.js]: - [ ] Kindly don't disable linters. If it isi giving error please fix it.

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS';
 
+// eslint-disable-next-line default-param-last
```
Comment (ahmetbozaci) [inline src/redux/categories/categories.js]: - [ ] Kindly don't disable linters. If it isi giving error please fix it.

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @sajid-munawar  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- You added redux according to requirement.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by roshan-bajgain — APPROVED** [review]
Hi @sajid-munawar, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sajid-munawar** [comment]
@roshan-bajgain Thanks bro, Can I contact you on slack, if you don't mind, please?

**Comment by roshan-bajgain** [comment]
yea bro, You can. Roshan Bajgain

**Comment by sajid-munawar** [comment]
@roshan-bajgain Please check

**Pull request merged**